packages/keyring-core/test/agent.ts

```diff
@@ -0,0 +1,41 @@
+import test from 'ava'
+import { JSDOM } from 'jsdom'
+import 'fake-indexeddb/auto'
+
+import { createAgent } from '../src/index.js'
```

Comment:
we can get rid of these extensions if we use `[--experimental-specifier-resolution=node](https://nodejs.org/api/esm.html#customizing-esm-specifier-resolution-algorithm)` option mentioned in https://github.com/avajs/ava/blob/main/docs/recipes/typescript.md#for-packages-with-type-module - I saw this earlier this week poking around at AVA but didn't get as far as you have here and am delighted to see it works locally. I'll push a commit doing this, happy to back it out if we don't want to use this